log/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+use lv2_core::feature::*;
+//use std::ffi::CString;
+use std::fmt;
+use std::os::raw::*; //get all common c_type
+use urid::*;
+
+pub struct EntryClass;
```

Comment:
I think having the entry types in a submodule (`entries` for instance) would be a cleaner, although it's not a lot of code, it'll show up as quite a few types in the rustdoc and might be a bit overwhelming for new users that might want to focus elsewhere (mainly the `Log` feature, or the entry type trait).